### ISSUE TITLE
NAV2018 code no longer works in BC

### ISSUE THREAD

**Issue by mynhardt7** [microsoft/AL#4396]
**Describe the bug**

The code below works in NAV 2018 CU8. When trying the code on BC CU2 it compiles, but the changes are not reflected in the UI.

**Screenshots**

![image](https://user-images.githubusercontent.com/41987377/50285336-971cd280-0464-11e9-9aa9-85a299d77fde.png)

![image](https://user-images.githubusercontent.com/41987377/50285365-af8ced00-0464-11e9-99c0-acdef76f9f6f.png)

**Versions:**

 - AL Langauge: 2.1.69331

 - Business Central: On-prem W1 13.2 (13.0.26413) CU2

**Comment by mynhardt7** [comment]
@atoader The caption should be changed. The **moveafter** statement should also have the effect of moving the County field which it doesn't.

**Comment by OleTeichmann** [comment]
I think you have to use a pagecustomization Object instead of a pageextension.

**Comment by JuricaB** [comment]
County caption is now handled via CaptionClass. For some reason this causes  explicit changes through extensions to be ignored. 

If you set County Name field in Country/Region table to Area, you will get desired result with no extension needed.

**Comment by atoader** [comment]
@mynhardt7 can I close this issue?

**Comment by mynhardt7** [comment]
@atoader It still seems like something is not working correctly. The code to modify the **Sell-to County** is completely ignored. The field is not made visible.

![image](https://user-images.githubusercontent.com/41987377/50826049-55849680-1343-11e9-917d-da60ef5f6cef.png)

**Comment by thpeder** [comment]
Hi @mynhardt7,
It is probably because of the ApplicationArea, try modifying it to All.

**Comment by atoader** [comment]
@mynhardt7 has Thomas's suggestion helped unblock you?

**Comment by qutreson** [comment]
@mynhardt7 any update on this issue? Is this still blocking you or can I close this issue?

**Comment by mynhardt7** [comment]
@qutreson Please close. I'll recheck now that BC15 is launched and create a new issue if the issue still exists.

**Comment by qutreson** [comment]
@mynhardt7 perfect, thank you. If you create a new issue, please also add a link to this issue in the description.
